tpl/setup.tpl

```diff
@@ -0,0 +1,155 @@
+<!DOCTYPE html>
+<html>
+
+<head>
```

Comment:
Header and footer should be split out into their own TPLs and then referenced in the `setup.tpl` and `index.tpl`.